cmd/cerbos/repl/internal/repl_test.go

```diff
@@ -544,3 +545,133 @@ func loadConditionsFromPolicy(t *testing.T, path string) []*runtimev1.Condition
 
 	return conds
 }
+
+func TestComplete(t *testing.T) {
+
+	rpPath := filepath.Join(test.PathToDir(t, "store"), "resource_policies", "policy_01.yaml")
+	p := test.LoadPolicy(t, rpPath)
+	rp := p.PolicyType.(*policyv1.Policy_ResourcePolicy)
+
+	testRules := rp.ResourcePolicy.Rules
+	testRules = testRules[0:3]
+	testRulesProtos := make([]proto.Message, len(testRules))
+
+	//nolint:gosimple // seems to be a false report of possible copy
+	for i, r := range testRulesProtos {
```

Comment:
Is looping over `testRulesProtos` and assigning to `testRulesProtos` intentional?

Reply:
nope! It doesn't impact the test because the completion / input is only the the index. Though Interestingly, when I tried using an empty array, it was crashing, (hence loading the valid rules).